node-operators/network-node/tanssi/tanssi-node-docker.md

```diff
@@ -38,27 +36,21 @@ Running a node requires syncing with the Tanssi chain and storing its state.
 
 Run the following command to create the directory where your node will store the databases containing blocks and chain states:
 
-=== "Dancelight"
-
-    ```bash
-    mkdir /var/lib/dancelight-data
-    ```
+```bash
+mkdir /var/lib/tanssi-data
+```
```

Comment:
make this dancelight? so that we can add tanssi later and it is under its own folder?

I would keep the tabs also. Check the other snippets plz